charts/nextcloud/templates/deployment.yaml

```diff
@@ -329,17 +329,18 @@ spec:
       {{- if .Values.nextcloud.extraVolumes }}
 {{ toYaml .Values.nextcloud.extraVolumes | indent 6 }}
       {{- end }}
+      # this is the securityContext for ALL containers in the nextcloud pod
       securityContext:
       {{- if .Values.nextcloud.podSecurityContext }}
         {{- with .Values.nextcloud.podSecurityContext }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
       {{- else }}
-        {{- if .Values.nginx.enabled }}
-        # Will mount configuration files as www-data (id: 82) for nextcloud
+        {{- if regexMatch "alpine$" nextcloud.image }}
```

Comment:
Hmm I have no idea

Reply:
I see, but now I kind of question the purpose of this PR, because you were able to simply set the fsGroup before and it doesn't make much difference to set a bool value somewhere or an int value somewhere else. Then this PR is also a breaking change, so I don't know how much sense it really makes to add this functionality :/

Reply:
~~How about removing support for `Values.image.tag` and adding a helper function that puts together the tag from `Chart.AppVersion` and `Values.image.flavor`? Then we can completely rely on the flavor and users don't need to set two different parameters.~~

This would break custom images with different tags though, so that isn't really an option. Maybe still an idea worth considering?

Reply:
But you can override the fsGroup, right? Or am I missing something here?

Reply:
Ok right, then let's go forward with `image.alpine`. I really don't like this situation, but I guess there's nothing we can do much better here :/

Reply:
Yeah, I think that would be better.